tests/results/TextPositionTest.py

```diff
@@ -30,3 +40,58 @@ def test_properties(self):
         uut = TextPosition(8, 39)
         self.assertEqual(uut.line, 8)
         self.assertEqual(uut.column, 39)
+
+
+class TextPositionLineColTest(unittest.TestCase):
+
+    def test_from_position(self):
```

Comment:
It'll be more useful for someone modifying your function. If he breaks something, he'll see immediately that, for example he messed up something related with `create_object`.